classes/oc/i18n.php

```diff
@@ -76,7 +76,7 @@ public static function initialize($locale = NULL, $charset = 'utf-8', $domain =
          * and force using gettext
          */
         if(defined('LC_ALL'))
-            $locale_res = setlocale(LC_ALL, self::$locale);
+            $locale_res = setlocale(LC_ALL, self::$locale.'.'.self::$charset);
```

Comment:
sure this wont break anything else? I think we had this before but then we had other issues with locales since not all OS will include the charset after the locale.